docs/source/CONTRIBUTING.rst

```diff
@@ -1,7 +1,9 @@
 Contributing
 ============
 
-Firstly, thanks for thinking of contributing to ``pydropsonde``; we are hoping that you didn't land here by mistake. But even if you did, now that you are here why not help us out a bit with this package? There are different ways](#how-can-you-help-us) in which you can contribute to HALO-DROPS. To know how to get started, jump to [Development Workflow](#development-workflow).
+Firstly, thanks for thinking of contributing to ``pydropsonde``; we are hoping that you didn't land here by mistake. But even if you did, now that you are here why not help us out a bit with this package? There are `different ways <how-help-us_>`_ in which you can contribute to HALO-DROPS. To know how to get started, jump to :ref:`dev_workflow`.
```

Comment:
Not your fault, but here it still says HALO-DROPS, could you change this to pydropsonde as well?

Reply:
Oops! Missed that.